samples/subsys/fs/littlefs/Kconfig

```diff
@@ -17,6 +17,8 @@ choice
 config APP_LITTLEFS_STORAGE_FLASH
 	bool "Use flash memory backend"
 
+config APP_LITTLEFS_STORAGE_BLK_SDMMC
```

Comment:
I would not introduce new option but I would make build decision depends on that whether CONFIG_FS_LITTLEFS_BLK_DEV==y instead. This imply that APP_LITTLEFS_STORAGE_FLASH, APP_LITTLEFS_STORAGE_BLK_SDMMC property would be not needed (in source code additional intermediary defines might be usefully).

This will remove some redundancy when configuration need to select the application property and closely linked the littlefs subsystem property.